server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/plan/FragmentInstance.java

```diff
@@ -165,6 +176,8 @@ public ByteBuffer serializeToByteBuffer() {
         DataOutputStream outputStream = new DataOutputStream(publicBAOS)) {
       id.serialize(outputStream);
       fragment.serialize(outputStream);
+      ReadWriteIOUtils.write(
+          timeOut > 0 ? timeOut : config.getQueryTimeoutThreshold(), outputStream);
```

Comment:
put this into constrcutor, in local dispatch processing, fragment instance won't be serialized and then deserialized.

Reply:
fixed